lib/src/components/NxCheckbox/NxCheckbox.scss

```diff
@@ -18,3 +18,16 @@
     vertical-align: 4px;
   }
 }
+
+.nx-radio-checkbox__input:checked:disabled {
+  + .nx-radio-checkbox__control {
+    &.nx-checkbox__box {
```

Comment:
I think this part of the selector is unneeded

Reply:
> I think this part of the selector is unneeded

I agree. It has been changed. 
d42994f4e3659af2c4d7b6ff0ed7d969ab0a6579